docs/satellite-clone.md

```diff
@@ -82,7 +73,6 @@ On the target server:
      # cp satellite-clone-vars.sample.yml satellite-clone-vars.yml
    ```
 2. Place the backup files in `/backup` folder. If using a different folder, update `backup_dir` variable in `satellite-clone-vars.yml`.
-3. If you are cloning RHEL 6 backup data to a RHEL 7 machine, update the variable `rhel_migration` to true in `satellite-clone-vars.yml`.
```

Comment:
the numbers here are not automatic right, I assume you need to update them for the remaining list?

Reply:
They are automatically updated in the rendered markdown to be sequential, but I updated them anyways to keep it straightforward.